unix_integration/src/resolver.rs

```diff
@@ -993,6 +993,21 @@ where
                         self.set_cache_userpassword(token.uuid, &cred).await?;
                         Ok(AuthResult::Success { token })
                     }
+                    Ok((AuthResult::Next(req), AuthCacheAction::PasswordHashUpdate { cred })) => {
```

Comment:
Mostly true - it depends on the type of the MFA provider. In this case with device grant, we can never do offline auth because of the device code requirement. But with something like webauthn we *could* do offline auth since we could potentially cache the public key of the authenticator. But yes, in most cases we can't do offline auth with  MFA here since most MFA's rely on a secret/seed that we don't have access to.